float64 dutyLeft
float64 dutyRight